tests/clickhouse-test

```diff
@@ -174,14 +174,25 @@ def configure_testcase_args(args, case_file, suite_tmp_dir, stderr_file):
             return ''.join(random.choice(alphabet) for _ in range(length))
         database = 'test_{suffix}'.format(suffix=random_str())
 
-        with open(stderr_file, 'w') as stderr:
+    testcase_args.testcase_database = database
+
+    j2env.globals['database'] = database
```

Comment:
Maybe better pass database as `context` to `stream` function in `render_test_template` https://jinja.palletsprojects.com/en/3.0.x/api/#jinja2.Template.stream

Reply:
Indeed, thanks!